packages/auto_mappr/test/integration/try_convert_test.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:test/test.dart';
+
+import 'fixture/default_target.dart' as fixture;
```

Comment:
I would prefer to use its own fixture so it does not depend on changes to default target fixtures.

Reply:
Done